src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -640,7 +640,16 @@ namespace ZoneEquipmentManager {
 				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).HeatMassFlow = 0.0;
 				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).HeatZoneTemp = 0.0;
 				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).HeatZoneHumRat = 0.0;
-			}
+			} else {
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolLoad = 0.0;
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolMassFlow = 0.0;
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolZoneTemp = Node( ZoneNode ).Temp;
+				CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolZoneHumRat = ZoneSizingInput( ControlledZoneNum ).CoolDesHumRat;
```

Comment:
@Nigusse @rraustad This line is failing with an array bounds error (and I suspect line 651 will also) as reported here
https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-f138d068e3cbcaddbdc9d8a5d688de9716021344-x86_64-Linux-Ubuntu-14.04-gcc-4.8-IntegrationCoverage-Debug.html
This was failing before I merged in develop and made the row-major updates.  It concerns me that this spot in the code would be looking back at ZoneSizingInput.  Seems there should be a different place to get this from, but I'm not following the logic here completely.  You can debug this by running just about any example file in the debugger (I ran 5ZoneAirCooled) and it failed on this line.